src/appengine/handlers/testcase_detail/testcase_variants.py

```diff
@@ -54,12 +54,17 @@ def _display_status(status):
       }
       if not is_pending:
         item.update({
-            'revision': variant.revision,
-            'crashType': variant.crash_type,
+            'revision':
+                variant.revision,
+            'crashType':
+                variant.crash_type,
             'crashStateLines': (variant.crash_state or '').strip().splitlines(),
```

Comment:
Line 59 now seems inconsistent with the other lines.
Why change them?

Reply:
Weird pyformat thing, can look later why.